player.go

```diff
@@ -0,0 +1,493 @@
+	}
+	var result PlayerDeviceList
+	err = json.NewDecoder(resp.Body).Decode(&result)
+	if err != nil {
+		return nil, err
+	}
+	return &result, nil
+}
+
+// PlayerState gets information about the playing state for the current user
+// This call requires authorization.
+//
+// Requires the ScopeUserReadPlaybackState scope in order to read information
+func (c *Client) PlayerState() (*PlayerState, error) {
+	return c.PlayerStateOpt(nil)
+}
+
+// PlayerStateOpt is like PlayerState, but it accepts additional
+// options for sorting and filtering the results.
+func (c *Client) PlayerStateOpt(opt *Options) (*PlayerState, error) {
+	spotifyURL := baseAddress + "me/player"
+	if opt != nil {
+		v := url.Values{}
+		if opt.Country != nil {
+			v.Set("market", *opt.Country)
+		}
+		if params := v.Encode(); params != "" {
+			spotifyURL += "?" + params
+		}
+	}
+	resp, err := c.http.Get(spotifyURL)
+	if err != nil {
+		return nil, err
+	}
+	defer resp.Body.Close()
+	if resp.StatusCode != http.StatusOK {
+		return nil, decodeError(resp.Body)
+	}
+	var result PlayerState
+	err = json.NewDecoder(resp.Body).Decode(&result)
+	if err != nil {
+		return nil, err
+	}
+	return &result, nil
+}
+
+// PlayerCurrentlyPlaying gets information about the currently playing status
+// for the current user.  This call requires authorization.
+//
+// Requires the ScopeUserReadCurrentlyPlaying scope or the ScopeUserReadPlaybackState scope
+// in order to read information
+func (c *Client) PlayerCurrentlyPlaying() (*CurrentlyPlaying, error) {
+	return c.PlayerCurrentlyPlayingOpt(nil)
+}
+
+// PlayerCurrentlyPlaying is like PlayerCurrentlyPlaying, but it accepts additional
+// options for sorting and filtering the results.
+func (c *Client) PlayerCurrentlyPlayingOpt(opt *Options) (*CurrentlyPlaying, error) {
+	spotifyURL := baseAddress + "me/player/currently-playing"
+	if opt != nil {
+		v := url.Values{}
+		if opt.Country != nil {
+			v.Set("market", *opt.Country)
+		}
+		if params := v.Encode(); params != "" {
+			spotifyURL += "?" + params
+		}
+	}
+	resp, err := c.http.Get(spotifyURL)
+	if err != nil {
+		return nil, err
+	}
+	defer resp.Body.Close()
+	if resp.StatusCode != http.StatusOK {
+		return nil, decodeError(resp.Body)
+	}
+	var result CurrentlyPlaying
+	err = json.NewDecoder(resp.Body).Decode(&result)
+	if err != nil {
+		return nil, err
+	}
+	return &result, nil
+}
+
+// TransferPlayback transfers playback to a new device and determine if
+// it should start playing. This call requires authorization.
+//
+// Note that a value of false for the play parameter when also transferring
+// to another device_id will not pause playback. To ensure that playback is
+// paused on the new device you should send a pause command to the currently
+// active device before transferring to the new device_id.
+//
+// Requires the ScopeUserModifyPlaybackState in order to modify the player state
+func (c *Client) TransferPlayback(device_ids []string, play bool) error {
```

Comment:
Don't use underscores, call the argument `deviceIDs`.

Again, are these strings or Spotify IDs?